Scrap the BrainyQuotes site

Rits1272 opened the pull request:
This file will get you the quote of the day.
This file can get you all quotes of the author given category.
This file can give a random quote given author and category.

Rits1272 committed:
Scrap the BrainyQuotes site

This file will get you the quote of the day.
This file can get you all quotes of the author given category.
This file can give a random quote given author and category.

akshatkarani commented:
You shouldn't change this file. Make a new file.  See [this](https://github.com/oss2019/pyquotes/pull/2) PR for reference.

Rits1272 commented:
Ok

Rits1272 committed:
Scraps the Brainy Quote site.

This file will get you the quote of the day.
This file can get you all quotes of the author given category.
This file can give a random quote given author and category.

Rits1272 commented:
This PR had conflicts with the base file.Therefore adding new file.
